src/services/serviceUtils.ts

```
@@ -43,7 +43,10 @@ const SERVICE_ID_REGEX =
 /**
  * Return the registry ids of services supported by a JSON Schema field definition
  */
-export function extractServiceIds(schema: Schema): RegistryId[] {
+export function extractServiceIds(
+  schema: Schema,
+  suppressError?: boolean
```

- Flags should always be passed via as options object for readability
- Method needs to document what "suppressError" does. Ideally we'd pick a better name that indicates what error is being suppressed